### PR TITLE
fix: use `_onMessage`

### PR DESCRIPTION
**Pull request by jrandolf-google**

### CODE REVIEW

File: src/common/Connection.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export class Connection extends EventEmitter {
     this.#delay = delay;
 
     this.#transport = transport;
-    this.#transport.onmessage = this.onMessage.bind(this);
+    this.#transport.onmessage = this._onMessage.bind(this);
```
Comment (OrKoN) [inline src/common/Connection.ts:69]: Should it start with underscore when it's a protected method? I think underscore is used to indicate private attributes?

Comment (jrandolf) [reply src/common/Connection.ts:69]: I'd prefer no underscore, but internal uses underscore hence the reason.

Comment (OrKoN) [reply src/common/Connection.ts:69]: I think we can just update the internal?

Comment (jrandolf) [reply src/common/Connection.ts:69]: +1

**Pull request closed without merging**